docs/docs/guides/web3_migration_guide.md

```diff
@@ -63,3 +63,21 @@ batch.execute();
 // in 4.x
 const responses = await batch.execute();
 ```
+
+### Web3-eth-accounts
+
+#### create and wallet.create
+
```

Comment:
Maybe we need to move this part to a separate guide? As we will have a separate for eth package.